.kitchen.yml

```diff
@@ -34,4 +34,6 @@ provisioner:
 
 suites:
   - name: default
-    run_list: recipe[fb_init_sample]
+    run_list:
```

Comment:
oh yes name and run_list are both at the same level inside the list (list item start represented by the '-') https://en.wikipedia.org/wiki/YAML#Basic_components_of_YAML

In Json it would be something like:

`[{name: 'default',run_list: ['recipe[fb_init_sample]']}]`